From b62c8292d223c8e7eafb3cfd2e997cbb63d15e23 Mon Sep 17 00:00:00 2001
From: Bart Van Assche <bvanassche@acm.org>
Date: Thu, 2 Feb 2023 14:01:42 -0800
Subject: scsi: ufs: core: Fix kernel-doc syntax

Fix the following kernel-doc warnings:

drivers/ufs/core/ufs-mcq.c:87: warning: Function parameter or member 'hba' not described in 'ufshcd_mcq_config_mac'
drivers/ufs/core/ufs-mcq.c:87: warning: Function parameter or member 'max_active_cmds' not described in 'ufshcd_mcq_config_mac'
drivers/ufs/core/ufs-mcq.c:107: warning: Function parameter or member 'hba' not described in 'ufshcd_mcq_req_to_hwq'
drivers/ufs/core/ufs-mcq.c:107: warning: Function parameter or member 'req' not described in 'ufshcd_mcq_req_to_hwq'
drivers/ufs/core/ufs-mcq.c:128: warning: Function parameter or member 'hba' not described in 'ufshcd_mcq_decide_queue_depth'

Link: https://lore.kernel.org/r/20230202220155.561115-1-bvanassche@acm.org
Fixes: 854f84e7feeb ("scsi: ufs: core: mcq: Find hardware queue to queue request")
Fixes: 2468da61ea09 ("scsi: ufs: core: mcq: Configure operation and runtime interface")
Fixes: 7224c806876e ("scsi: ufs: core: mcq: Calculate queue depth")
Cc: Asutosh Das <quic_asutoshd@quicinc.com>
Signed-off-by: Bart Van Assche <bvanassche@acm.org>
Reviewed-by: Asutosh Das <quic_asutoshd@quicinc.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
---
 drivers/ufs/core/ufs-mcq.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/ufs/core/ufs-mcq.c b/drivers/ufs/core/ufs-mcq.c
index dd476f9e797cc..31df052fbc417 100644
--- a/drivers/ufs/core/ufs-mcq.c
+++ b/drivers/ufs/core/ufs-mcq.c
@@ -77,8 +77,8 @@ MODULE_PARM_DESC(poll_queues,
 
 /**
  * ufshcd_mcq_config_mac - Set the #Max Activ Cmds.
- * @hba - per adapter instance
- * @max_active_cmds - maximum # of active commands to the device at any time.
+ * @hba: per adapter instance
+ * @max_active_cmds: maximum # of active commands to the device at any time.
  *
  * The controller won't send more than the max_active_cmds to the device at
  * any time.
@@ -96,8 +96,8 @@ void ufshcd_mcq_config_mac(struct ufs_hba *hba, u32 max_active_cmds)
 /**
  * ufshcd_mcq_req_to_hwq - find the hardware queue on which the
  * request would be issued.
- * @hba - per adapter instance
- * @req - pointer to the request to be issued
+ * @hba: per adapter instance
+ * @req: pointer to the request to be issued
  *
  * Returns the hardware queue instance on which the request would
  * be queued.
@@ -114,7 +114,7 @@ struct ufs_hw_queue *ufshcd_mcq_req_to_hwq(struct ufs_hba *hba,
 
 /**
  * ufshcd_mcq_decide_queue_depth - decide the queue depth
- * @hba - per adapter instance
+ * @hba: per adapter instance
  *
  * Returns queue-depth on success, non-zero on error
  *
-- 
cgit 

